scripts/diff_studies.py

```diff
@@ -0,0 +1,200 @@
+"""
+Script to check a study on pedcbioportal for differences against a local build
+"""
+
+import argparse
+from bravado.client import SwaggerClient
+from bravado.requests_client import RequestsClient
+from urllib.parse import urlparse
+
+
+def clinical_diffs(portal, build, portal_attr, build_attr, clin_type, out):
+    """
+    Compare differences in portal sample data and build.
+    """
+    # gross ID diffs
+    portal_clinical_ids = set(portal.keys())
+    build_clinical_ids = set(build.keys())
+    portal_only = list(portal_clinical_ids - build_clinical_ids)
+    portal_only.sort()
```

Comment:
sorted will return return a sorted list from a set
```suggestion
    portal_only = sorted(portal_clinical_ids - build_clinical_ids)
```

Reply:
Ah, ok, yeah, for some reason I forgot about it...